gecko/glue/include/mozilla-config-x86_64-linux.h

```diff
@@ -136,22 +134,26 @@
 #define MOZ_LOGGING 1
 #define MOZ_MACBUNDLE_ID org.mozilla.nightlydebug
 #define MOZ_MEMORY 1
-#define MOZ_MEMORY_LINUX 0
+#define MOZ_PEERCONNECTION 1
 #define MOZ_PERMISSIONS 1
 #define MOZ_PHOENIX 1
 #define MOZ_PLACES 1
 #define MOZ_PROFILING 1
 #define MOZ_PULSEAUDIO 1
+#define MOZ_RAW 1
 #define MOZ_REFLOW_PERF 1
 #define MOZ_REFLOW_PERF_DSP 1
-#define MOZ_REPLACE_MALLOC 0
+#define MOZ_REPLACE_MALLOC 1
```

Comment:
Given that we're undefing the things we want to turn off, maybe we should import this instead of making a copy into glue?